src/main/resources/META-INF/maven/site-macros.vm

```diff
@@ -111,7 +111,7 @@
 #**##link ( $link $null true )
 #end
 ##
-#macro( image $image $class )
```

Comment:
look like `$class` has special value ... 😄

Reply:
You in Velocity?

Reply:
I don't know where ... but when I change variable name it is ok ...